greatNovels.sql

```diff
@@ -0,0 +1,161 @@
+CREATE DATABASE greatNovels;
+
+  USE greatNovels;
+
+  CREATE TABLE authors (
+   id INT auto_increment,
+   nameFirst VARCHAR(255),
+   nameLast VARCHAR(255),
+   createdAt DATETIME DEFAULT NOW(),
+   updatedAt DATETIME DEFAULT NOW() on UPDATE NOW(),
+   deletedAt DATETIME,
+   PRIMARY KEY(id)
+ );
+
+  CREATE TABLE novels (
+   id INT auto_increment,
+   title VARCHAR(255),
+   authorId INT,
+   createdAt DATETIME DEFAULT NOW(),
+   updatedAt DATETIME DEFAULT NOW() on UPDATE NOW(),
+   deletedAt DATETIME,
+   PRIMARY KEY(id),
+   FOREIGN KEY(authorId) REFERENCES authors(id)
+ );
+
+  CREATE TABLE genres (
+   id INT auto_increment,
+   name VARCHAR(255),
+   createdAt DATETIME DEFAULT NOW(),
+   updatedAt DATETIME DEFAULT NOW() on UPDATE NOW(),
+   deletedAt DATETIME,
+   PRIMARY KEY(id)
+ );
+
+  CREATE TABLE novelsGenres (
+   genreId INT,
+   novelId INT,
+   createdAt DATETIME DEFAULT NOW(),
+   updatedAt DATETIME DEFAULT NOW() on UPDATE NOW(),
+   deletedAt DATETIME,
+   PRIMARY KEY (genreId, novelId)
```

Comment:
No foreign key setup here so these columns aren't actually connected to the other table the way they should be.